apps/jobs/views.py

```diff
@@ -420,10 +420,12 @@ def get_remaining_submissions(request, challenge_phase_pk, challenge_pk):
     max_submissions_per_month_count = challenge_phase.max_submissions_per_month
     max_submissions_per_day_count = challenge_phase.max_submissions_per_day
 
+    submission_status_to_exclude = [Submission.FAILED, Submission.CANCELLED]
```

Comment:
It would be great if this can come from some utils function so that we don't have to write this at multiple places. Does that make sense?